cfme/tests/infrastructure/test_infra_quota.py

```diff
@@ -129,7 +129,7 @@ def _provisioner(template, provisioning_data, delayed=None):
     return _provisioner
 
 
-@pytest.mark.meta(blockers=[1287102])
+@pytest.mark.uncollectif(lambda: version.current_version() >= '5.5')
 def test_group_quota_max_memory_check_by_tagging(
```

Comment:
My only question here, is that it clearly says ```test_group``` and not ```test_tenant```. I didn't delve in, but is this test for testing group quota or tenant quota?